ajax/libs/datatables-fixedheader/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "datatables-fixedheader",
-  "version": "2.1.1",
-  "filename": "dataTables.fixedHeader.min.js",
+  "version": "3.1.3",
+  "filename": "js/dataTables.fixedHeader.min.js",
```

Comment:
Not sure if this change should be in the commit involving "auto-update" update because filename's path is the relative path to `basePath` in auto-update config.